### ISSUE TITLE
Edit a row with double click

### ISSUE THREAD

**Issue by gregdusoli** [akveo/ng2-smart-table#670]
Hi, dudes!

Is there anyway to enter in edit mode of a row with double click, rather than using the edit button on Actions column (coz I've choose to hide the default action buttons)?

**Comment by aguarino77** [comment]
I agree, it would be nice to enter the edit and create modes programmatically

**Comment by josephbagnes** [comment]
+1 for this feature please

**Comment by gbarre** [comment]
Any news about this feature ?
